plugin/src/ios/CBTBrowserTab.h

```diff
@@ -18,8 +18,14 @@
 #import <SafariServices/SafariServices.h>
 
 @interface CBTBrowserTab : CDVPlugin
+{
+    @protected
+    SFSafariViewController* _safari;
+}
 
 - (void) isAvailable:(CDVInvokedUrlCommand*)command;
 - (void) openUrl:(CDVInvokedUrlCommand*)command;
+- (void) close:(CDVInvokedUrlCommand*)command;
```

Comment:
Add a space before the asterisk. `- (void) close:(CDVInvokedUrlCommand *)command;` (can you update the two previous lines as well?).